templates/support/SecurityClientController.swift.hbs

```diff
@@ -0,0 +1,41 @@
+//  
+//  {{>frag/generatedBy}}
+//
+
+import Foundation
+
+/**
+ Delegates security requirements to the appropriate SecurityClient. 
+
+ This controller serves as the default intended security client for any API configuration and can be safely shared by multiple API instances.
+
+ - Note: Specific `SecurityClient` objects can be inserted or replaced dynamically to accommodate user behavior changes, such as signing in or signing out.
+*/
+public class SecurityClientController: SecurityClient {
```

Comment:
This name is feeling quite generic now given that most people simply might not use it... as most APIs have a single scheme. Perhaps the name should be `MultiSecurityClientSupport` even.

Reply:
Discussed internally, we'll keep it as is